### PR TITLE
Hotfix: Run CI after every push

### PR DESCRIPTION
**Pull request by garciparedes**
As we said today, during the daily meeting, we need to update the Github Actions config to run the tests workflow after each new change on any branch (so I removed the `branches` constraint).

### CODE REVIEW

File: .github/workflows/python-tests.yml [hunk 1]
```diff
@@ -1,6 +1,5 @@
 on:
   push:
-    branches: [ main ]
   pull_request:
```
Comment (garciparedes) [inline .github/workflows/python-tests.yml:3]: After doing that, I've doubt, are there any cases in which the `pull_request` event triggers but not the `push` one? I mean, maybe with the "on push" event we are able to cover both cases.

**Comment by garciparedes** [comment]
This hot fix is related to https://github.com/Clariteia/minos-pypackage/issues/1

File: .github/workflows/python-tests.yml (hunk 1 above)
Comment (andrea-mucci) [reply .github/workflows/python-tests.yml:5]: > ne? I mean, maybe with the "on push" event we are able to cover both case

Yes, have sense

we reduce to "on push" trigger

Comment (andrea-mucci) [reply .github/workflows/python-tests.yml:5]: I will invalidate this pull request and you put a new one

**Review by andrea-mucci — CHANGES_REQUESTED** [review]
Trigger only on push, an remove the branches trigger

File: .github/workflows/python-tests.yml (hunk 1 above)
Comment (garciparedes) [reply .github/workflows/python-tests.yml:5]: Okay @andrea-mucci, I'll close this one and create an new one 🙂

**Pull request closed without merging**